src/coreclr/jit/importercalls.cpp

```diff
@@ -2625,6 +2606,28 @@ GenTree* Compiler::impIntrinsic(GenTree*                newobjThis,
         }
     }
 
+    // Intrinsics that we should make every effort to expand for NativeAOT.
+    // If the intrinsic cannot possibly be expanded, it's fine, but
+    // if it can be, it should expand.
+    switch (ni)
+    {
+        // CreateSpan must be expanded for NativeAOT
+        case NI_System_Runtime_CompilerServices_RuntimeHelpers_CreateSpan:
+        case NI_System_Runtime_CompilerServices_RuntimeHelpers_InitializeArray:
+            betterToExpand |= IsTargetAbi(CORINFO_NATIVEAOT_ABI);
+            break;
+
+        case NI_Internal_Runtime_MethodTable_Of:
+        case NI_System_Activator_AllocatorOf:
+        case NI_System_Activator_DefaultConstructorOf:
+        case NI_System_EETypePtr_EETypePtrOf:
+            betterToExpand = true;
+            break;
+
+        default:
+            break;
+    }
```

Comment:
```suggestion
    if (IsTargetAbi(CORINFO_NATIVEAOT_ABI))
    {
        // Intrinsics that we should make every effort to expand for NativeAOT.
        // If the intrinsic cannot possibly be expanded, it's fine, but
        // if it can be, it should expand.
        switch (ni)
        {
            // CreateSpan must be expanded for NativeAOT
            case NI_System_Runtime_CompilerServices_RuntimeHelpers_CreateSpan:
            case NI_System_Runtime_CompilerServices_RuntimeHelpers_InitializeArray:
            case NI_Internal_Runtime_MethodTable_Of:
            case NI_System_Activator_AllocatorOf:
            case NI_System_Activator_DefaultConstructorOf:
            case NI_System_EETypePtr_EETypePtrOf:
                betterToExpand = true;
                break;

            default:
                break;
        }
```
All these are native AOT specific as the comment says. We can check for it upfront to save a few instructions for `!IsTargetAbi(CORINFO_NATIVEAOT_ABI)`.

Reply:
I didn't want to change it so that I don't have to deal with the formatter, but we can do this...